### PR TITLE
Update to streams RC4

### PR DESCRIPTION
**Pull request by 2m**

### CODE REVIEW

**Comment by 2m** [comment]
This will fail until RC4 artifacts are promoted.

File: src/main/scala/sample/stream/GroupLogFile.scala [hunk 1]
```diff
@@ -27,7 +28,8 @@ object GroupLogFile {
     import akka.stream.io.Implicits._ // add file sources to Source object or use explicitly: SynchronousFileSource(f)
     Source.synchronousFile(logFile).
       // parse chunks of bytes into lines
-      transform(() => parseLines(System.lineSeparator, 512)).
+      via(Framing.delimiter(ByteString(System.lineSeparator), 512, allowTruncation = true)).
```
Comment (drewhk) [inline src/main/scala/sample/stream/GroupLogFile.scala]: name the "512" parameter

**Comment by rkuhn** [comment]
LGTM

File: src/main/scala/sample/stream/GroupLogFile.scala (hunk 1 above)
Comment (rkuhn) [inline src/main/scala/sample/stream/GroupLogFile.scala]: agreed

Comment (2m) [inline src/main/scala/sample/stream/GroupLogFile.scala]: :+1:

**Comment by drewhk** [comment]
Doesn' this also need updates to the tutorial text because of the Framing?

**Comment by 2m** [comment]
No, parsing was not mentioned in the tutorial.

**Pull request merged**